odoo/addons/base/transpiler/local_test_transpiler.py

```diff
@@ -0,0 +1,43 @@
+import glob
```

Comment:
Definitively not the right way to test.
Please write a regular test case, at least it will be reported correctly on the runbot.

Reply:
Maybe the js code and the test itself should move to module `test_assetsbundle`.
The purpose of this module is to host such test code.